index.html

```diff
@@ -8,9 +8,9 @@
     <link rel="stylesheet" href="./style.css">
     <title>Software Developer Portfolio Christian Momo</title>
 </head>
-<body>
-    <div class ="header_headline_container">
-        <header>
+<body class="flex-column">
+    <div class ="header_headline_container flex-column">
+        <header class="flex-row">
```

Comment:
- [ ] [OPTIONAL] Excellent job implementing the previous reviewer's suggestion 🥇 I think it would be better to give `header` some `z-index` value so that the header is always in front 💪 To learn more about it, please follow the below mentioned articles 💡
- [z-index by mozilla](https://developer.mozilla.org/en-US/docs/Web/CSS/z-index)
- [z-index by CSS tricks](https://css-tricks.com/almanac/properties/z/z-index/)